modules/engage-theodul-plugin-tab-shortcuts/src/main/java/org/opencastproject/engage/theodul/plugin/tab/shortcuts/EngagePluginTabShortcuts.java

```diff
@@ -22,19 +22,40 @@
 package org.opencastproject.engage.theodul.plugin.tab.shortcuts;
 
 import org.opencastproject.engage.theodul.api.AbstractEngagePlugin;
+import org.opencastproject.engage.theodul.api.EngagePlugin;
+import org.opencastproject.engage.theodul.api.EngagePluginManager;
 
 import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import javax.ws.rs.Path;
 
 @Path("/")
+@Component(
+    property = {
+        "opencast.engage.plugin.name=EngagePluginTabShortcuts",
+        "opencast.engage.plugin.shortcuts=Simple implementation of a tab with the shortcuts",
```

Comment:
This should be `opencast.engage.plugin.description` (wrong in the original XML).

Reply:
fixed